wordpress-mobile/WordPress-iOS
# Remove Giphy code

Ref #13803 

- Shared PR: https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/264 (to remove Tracks)
- Kit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/249 (just to update Shared pod)
- Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/301 (just to update Shared & Kit pods)

(Sorry about all the pod PRs. Shared is the only one with actual changes, but I had to update the other two to satisfy dependencies.)

This removes all Giphy related code. The only remnants should be localized strings, which should get removed when translations are updated.

To test:
Giphy has been disabled for quite a while, so there should be no visible changes. So just make sure other media is still functional.

- Verify the app builds and runs with no Giphy related warnings/errors.
- Verify media import via `Free Photo Library`.
- Verify media import via  `Free GIF Library`.

PR submission checklist:
